otx/cli/manager/config_manager.py

```diff
@@ -437,6 +437,7 @@ def build_workspace(self, new_workspace_path: Optional[str] = None) -> None:
         # Copy config files
         config_files = [
             (model_dir, "model.py", train_type_dir),
+            (model_dir, "model_multilabel.py", train_type_dir),
```

Comment:
We modified it to import `model_multilabel.py` based on model dir in this line, then we can delete line 446, right?

Reply:
I guess yes, you're right